### DISCUSSION TITLE
Handle XSS (Cross Site Scripting) in input fields and text area

### DISCUSSION THREAD

**Discussion by awnish-npg** [directus/directus#7218] [Archive]
Is there was to handle the
The issue occurs because the browser interprets the input as active HTML, JavaScript or VBScript. To avoid this, output should be encoded according to the output location and context.
Another option could be: Do not allow HTML file to be uploaded to webserver.

![image](https://user-images.githubusercontent.com/60690093/128359131-39e431fd-b89f-4ad4-8a02-8f3013343235.png)

**Comment by rijkvanzanten** [comment]
For any issues in Directus, please open a (detailed) [Issue](https://github.com/directus/directus/issues/new?assignees=&labels=Bug+%28Potential%29&template=bug_report.yml). For any security related problems, email security@directus.io directly (as per our security guidelines: https://github.com/directus/directus/blob/main/security.md)
